aten/src/ATen/native/Cross.cpp

```diff
@@ -6,7 +6,29 @@
 
 #include <ATen/native/Cross.h>
 
-namespace at { namespace native {
+namespace at {
+namespace meta {
+
+TORCH_PRECOMPUTE_META_FUNC(linalg_cross)
+(const Tensor & input, const Tensor & other, const int64_t dimension) {
+  const Tensor& out = maybe_get_output(0);
+
+  auto device1 = input.device().type();
+  auto device2 = other.device().type();
```

Comment:
looks like `out` `device` and `device2` are unused (internally tests complaining)

Reply:
Oops, sorry! This should be removed now. Thanks for pointing it out, Brian.